app/models/district.rb

```diff
@@ -13,6 +13,7 @@ class District < ActiveRecord::Base
   belongs_to :state
   has_and_belongs_to_many :zip_codes
   has_and_belongs_to_many :campaigns
+  has_one :legislator#, dependent: :delete
```

Comment:
There may not be so much value in this, but you might consider naming these to `has_one :representative, class_name: "Legislator`"  Feel free to ignore.